### PR TITLE
Set Up Multiple Environment Settings #155269107

### PR DESCRIPTION
**Pull request by wanjiku-carol**
**What does this PR do?**
Creates multiple environment setup options

**Description of work done?**
- Create settings folder
- Separate files for different environments
- Environmental settings

**How can this be manually tested?**
> $ APP_ENV=(environment) python manage.py runserver

**Relevant pivotal tracker stories?**
[#155269107](https://www.pivotaltracker.com/story/show/155269107)

### CODE REVIEW

File: art/settings/base.py [hunk 1]
```diff
@@ -37,15 +37,17 @@
 SECRET_KEY = config('SECRET_KEY')
 DEBUG = config('DEBUG', default=False, cast=bool)
```
Comment (egichuri) [inline art/settings/base.py]: we can remove this debug since it's in the env specific settings

File: art/settings/base.py [hunk 2]
```diff
@@ -37,15 +37,17 @@
...
 DATABASES = {
-    'default': dj_database_url.config()
+    'default': dj_database_url.config(
+        default=config('DATABASE_URL')
+    )
 }
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
```
Comment (egichuri) [inline art/settings/base.py]: this too

File: art/settings/base.py [hunk 3]
```diff
@@ -37,15 +37,17 @@
...
 
 ALLOWED_HOSTS = [
```
Comment (egichuri) [inline art/settings/base.py]: this should be env specific - `art-backend.herokuapp.com` for prod and `'127.0.0.1',  '0.0.0.0',` for dev

Comment (wanjiku-carol) [reply art/settings/base.py]: Should I have them removed from `base.py` when I place them in the other envs?

Comment (egichuri) [reply art/settings/base.py]: yea.. the values in prod or dev envs should not be in base

**Pull request merged**